keyboards/boardsource/3x4/readme.md

```diff
@@ -0,0 +1,14 @@
+# 3x4
+![Boardsource 3x4 macro](https://i.imgur.com/EiPF1hh.jpg)
+
+this macro pad / small 12 key was inspired by the plaid look
+
+Keyboard Maintainer: [Boardsource](https://github.com/daysgobye)  
+Hardware Supported: 3x4 v1
+Hardware Availability: this is macro pad is available from the Boardsource store
```

Comment:
Please add a link.

Reply:
Ok I did it. Thank you very much and good catch.